SpatialGDK/Documentation/content/toolbars.md

```diff
@@ -95,15 +95,33 @@ The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS GDK fo
 
  ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/toolbar-settings.png)
 
-##### Configuration
+##### General
 
 | Setting | Description |
 | --- | --- |
 | SpatialOS directory | If you're using a non-standard structure, you'll need to set this yourself. This is empty by default. If you leave it empty, it defaults to `<GameRoot>/../spatial`. |
+
+##### Play In Editor Settings
+
+| Setting | Description |
+| --- | --- |
+| Delete dynamically spawned entities | If checked, the GDK deletes any dynamically spawned entities from your game world when you end the PIE session. |
```

Comment:
"From your SpatialOS deployment" is the better wording, right?

Reply:
Yep, updating now